src/test/java/wordsWithDoubleLetters/WordsWithDoubleLettersTest.java

```diff
@@ -0,0 +1,50 @@
+package wordsWithDoubleLetters;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class WordsWithDoubleLettersTest {
+    private final WordsWithDoubleLetters words = new WordsWithDoubleLetters();
+
+    @Test
+    void empty() {
+        Assertions.assertThat(words.getWords(""))
+                .isEqualTo("");
+    }
+
+    @Test
+    void blank() {
+        Assertions.assertThat(words.getWords(" "))
+                .isEqualTo("");
+    }
+
+    @Test
+    void wordWithDoubleLetters() {
+        Assertions.assertThat(words.getWords("Hello"))
+                .isEqualTo("");
+    }
+
+    @Test
+    void doubleLetters() {
+        Assertions.assertThat(words.getWords("ll"))
+                .isEqualTo("");
+    }
+
+    @Test
+    void wordsWithOnlyDoubleLetters() {
+        Assertions.assertThat(words.getWords("Hello  password"))
+                .isEqualTo("");
+    }
+
+    @Test
+    void noDoubleLetters() {
+        Assertions.assertThat(words.getWords("pasword word word Helo"))
+                .isEqualTo("pasword word word Helo");
+    }
+
+    @Test
+    void task() {
+        Assertions.assertThat(words.getWords("password wword word Heeeeeeello"))
```

Comment:
I believe, such a test will fail just because of extra spaces in result:
```
Assertions.assertThat(words.getWords("word1 wword wword word2")).isEqualTo("word1 word2");
```

Reply:
Another case that seems to fail:
```
Assertions.assertThat(words.getWords("woword")).isEqualTo("woword");
```

Reply:
Yes, because this regex is for double letters in a row. I will think about how to extend this solution.

Reply:
fixed